ROADMAP.md

```diff
@@ -173,22 +174,66 @@ Any dates listed below and the specific issues that will ship in a given milesto
-  * port [stack-minimal-aws](https://github.com/crossplane/stack-minimal-aws) and [stack-minimal-azure](https://github.com/crossplane/stack-minimal-azure) to Template Stacks
-
+* Security enhanced mode with `stack manage --restrict-core-apigroups`, which restricts packages from being installed with permissions on the core API group. [#1333 ](https://github.com/crossplane/crossplane/issues/1333)
 * Stacks Manager support for private repos and robot account credentials
 * Release process and efficiency improvements
 
+
+## v0.10.0
+* Backup/restore support - e.g. with Velero
+  - Allow a KubernetesApplication to be backed up and restored [crossplane#1382](https://crossplane/crossplane/issues/#1382)
+  - Allow connection secrets to be backed up and restored [crossplane-runtime#140](https://crossplane/crossplane-runtime/issues/#140)
+  - Support backup and restore of all GCP managed resources [provider-gcp#207](https://crossplane/provider-gcp/issues/#207)
+  - Support backup and restore of all Azure managed resources [provider-azure#128](https://crossplane/provider-azure/issues/#128)
+  - Support backup and restore of all AWS managed resources [provider-aws#181](https://crossplane/provider-gcp/issues/#181)
+  - Allow Stack, StackInstall, StackDefinition to be backed up and restored [crossplane#1389](https://crossplane/crossplane/issues/#1389)
+  - Backup and Restore doc [crossplane#1353](https://crossplane/crossplane/issues/#1353)
+
+* v1beta1 quality conformance doc [#933](https://github.com/crossplane/crossplane/issues/933)
+* v1beta1 quality for AWS API types  
+  - Networking and VPC [crossplane/provider-aws#145](https://github.com/crossplane/provider-aws/issues/145)
+
+* AWS Provider: additional API types [crossplane/provider-aws#149](https://github.com/crossplane/provider-aws/issues/149)
+  - DynamoDB [crossplane/provider-aws#147](https://github.com/crossplane/provider-aws/issues/147)
+  - SQS [crossplane/provider-aws#170](https://github.com/crossplane/provider-aws/issues/170)
+  - Cert Manager [crossplane/provider-aws#171](https://github.com/crossplane/provider-aws/issues/171)
+  - DNS [crossplane/provider-aws#172](https://github.com/crossplane/provider-aws/issues/172)
+
+* Basic versioning and upgrade support [#1334](https://github.com/crossplane/crossplane/issues/1334)
+
+* Resource composition - experimental MVP [#1343](https://github.com/crossplane/crossplane/issues/1343)
+
+* Experimental support for [OAM](https://oam.dev/) (Open Application Model) API types
+  * Revised [Kubernetes-friendly OAM spec](https://github.com/oam-dev/spec/pull/304/files)
+  * OAM App Config Controller support [#1268](https://github.com/crossplane/crossplane/issues/1268)
+  * Enhance Crossplane to support a choice of local and remote workload scheduling
+  * OAM sample app: [crossplane/app-service-tracker](https://github.com/crossplane/app-service-tracker)
+
+* Docs overhaul (part 2/2) - https://crossplane.github.io/docs
```

Comment:
any reason to use crossplane.github.io instead of https://crossplane.io/docs/?

Reply:
good catch, updated. thx!